react-native-purchases-ui/src/index.tsx

```diff
@@ -61,12 +65,12 @@ export default class RevenueCatUI {
    */
   public static PAYWALL_RESULT = PAYWALL_RESULT;
 
-  public static presentPaywall({}: PresentPaywallParams = {}): Promise<PAYWALL_RESULT> {
-    return RNPaywalls.presentPaywall();
+  public static presentPaywall(params: PresentPaywallParams): Promise<PAYWALL_RESULT> {
```

Comment:
Hmm is this a breaking change since now passing the params is mandatory and we don't have a default value?

Reply:
oh this was actually unintended! I broke it when rebasing. Thanks!